content/blog/dplyr-1-1-0-per-operation-grouping/index.Rmd

```diff
@@ -0,0 +1,191 @@
+---
+output: hugodown::hugo_document
+slug: dplyr-1-1-0-per-operation-grouping
+title: "dplyr 1.1.0: Per-operation grouping"
+date: 2023-02-01
+author: Davis Vaughan
+description: >
+    dplyr now supports an experimental per-operation grouping syntax. This serves as an
+    alternative to `group_by()` and always returns an ungrouped data frame, meaning that you
+    never need to remember to `ungroup()`.
+photo:
+  url: https://www.pexels.com/photo/fruit-stand-375897/
+  author: Clem Onojeghuo
+categories: [package] 
+tags: [dplyr]
```

Comment:
I just remembered we used a dplyr-1-0-0 tag (https://www.tidyverse.org/tags/dplyr-1-0-0/) for 1.0.0; we should do the same here, and you can add a link to each blog post to see all the posts.